### PR TITLE
Add separate README for chip8_vm

### LINKED ISSUE #6: Add screenshots to README
As with any kind of graphical project, people want to see how it looks like.
While there's not much to see except for the ROM specific output and a window border, I suggest we just add a screenshot of that.

We could either use the included `intro.ch8` or maybe something like `pong2.ch8`.

I can provide screenshots from my Linux box, but maybe OS X looks .. uhm .. better.

### PR DESCRIPTION
**Pull request by jakerr**
Add separate README for chip8_vm
Also update top-level README adding screenshot.

Fixes: #6, #7

### CODE REVIEW

File: chip8_vm/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+This crate implements a virtual machine of the CHIP-8 programming language. It
+can be used as the back end for CHIP-8 emulator projects / debuggers etc.
+
+Status
+==
+* All 35 original Chip-8 instructions are implemented.
+
+Usage
+==
+This crate only has no system library dependencies, only dependencies on other
```
Comment (robo9k) [inline chip8_vm/README.md]: Maybe rephrase as

> This crate does not depend on additional system libraries, it only depends on other Rust crates.

File: chip8_vm/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+rust crates. It can be used as a back end for your own emulator by adding it as
+a dependency in your `Cargo.toml`.
+
+To depend on the latest released version check [crates.io] (https://crates.io/crates/chip8_vm)
```
Comment (robo9k) [inline chip8_vm/README.md]: Cargo allows version wildcards, so in theory we could have `0.x.y` version crates and tell users to use

``` toml
[dependencies]
chip8_vm = "0.*"
```

This would mean they get the latest matching version whenever they update (`cargo update --package chip8_vm`).

Once Rust itself is somewhat stable, we could switch to `1.x.y` versions and tell users to use a specific version (and rely on `SemVer` compatibility).

File: chip8_vm/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+To depend on the development version you can add the follwing to your
+`Cargo.toml`
+
+```toml
+[dependencies.chip8_vm]
+git = "https://github.com/jakerr/chip8-rust"
+```
+
+See an exmaple integration with a UI in the [chip8_ui](https://github.com/jakerr/chip8-rust/blob/master/src/main.rs) crate.
```
Comment (robo9k) [inline chip8_vm/README.md]: > See an ex**am**ple integration

I can't count how often I misspelled that one myself :grinning:

File: chip8_vm/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Spec
+==
+These two resources were used as the spec for this vm:
+* [Cowgod's Chip-8 Technical Reference v1.0](http://devernay.free.fr/hacks/chip8/C8TECH10.HTM)
+* [Mastering Chip-8 By Matthew Mikolay](http://mattmik.com/chip8.html)
+They were both incredibly helpful so thanks to the authors!
```
Comment (robo9k) [inline chip8_vm/README.md]: Potentially add a newline so the note does not display right after the second bullet point but instead below.

File: chip8_vm/README.md (hunk 1 above)
Comment (jakerr) [inline chip8_vm/README.md]: Thanks that sentence made no sense :blush:

File: chip8_vm/README.md (hunk 3 above)
Comment (jakerr) [inline chip8_vm/README.md]: Done.

File: chip8_vm/README.md (hunk 4 above)
Comment (jakerr) [inline chip8_vm/README.md]: Done.

File: chip8_vm/README.md (hunk 2 above)
Comment (jakerr) [inline chip8_vm/README.md]: Done.

**Comment by jakerr** [comment]
All comments addressed, thanks for the review!
